textgrad/engine/ollama.py

```diff
@@ -0,0 +1,100 @@
+try:
+    from ollama import Client
+    # from openai import OpenAI
+except ImportError:
+    raise ImportError("If you'd like to use OpenAI models, please install the openai package by running `pip install openai`, and add 'OPENAI_API_KEY' to your environment variables.")
```

Comment:
Can we improve this error message? Let's either describe what to do in case ollama is not installed, or point folks to a resource that describes this.